gadgets/trace_dns/test/integration/trace_dns_test.go

```diff
@@ -68,15 +68,13 @@ func TestTraceDNS(t *testing.T) {
 	require.NoError(t, err, "new container factory")
 	serverContainerName := "test-trace-dns-server"
 	clientContainerName := "test-trace-dns-client"
-	// TODO: this should be configurable
-	serverImage := "ghcr.io/inspektor-gadget/dnstester:latest"
-	clientImage := "docker.io/library/busybox:latest"
-
+	
+	utils.InitDnsTest()
```

Comment:
I don't think we need to involve another package on this. Let's read the env variable directly on this package.